src/component/modal.js

```diff
@@ -34,15 +34,39 @@ const styles = StyleSheet.create({
   },
 });
 
-const Modal = ({ title = '', onClose, children, style }) => (
-  <View style={[styles.modal, style]}>
-    <H4Text style={styles.title}>{title.toUpperCase()}</H4Text>
-    <Button style={styles.cancelBtn} onPress={onClose}>
-      <Icon image="cancel-grey" style={styles.cancelIcon} />
-    </Button>
-    {children}
-  </View>
-);
+class Modal extends Component {
+  constructor(props) {
+    props.title = props.title || '';
+    super(props);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+  }
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyDown);
```

Comment:
We should check if `document` exist as this will crash on mobile.

Also please use `e => this.handleKeyDown(e)` instead of bind.